File: backend/src/main/kotlin/ua/nure/makieiev/ark/model/entity/FilterWorkLogStatistic.kt
package ua.nure.makieiev.ark.model.entity

import java.math.BigDecimal
import java.util.*

data class FilterWorkLogStatistic(var avgTemperature: String? = null,
                                  var sumFilterContamination: BigDecimal? = null,
                                  var scanDate: Date? = null)

File: frontend/ark-project/src/app/models/entity/FilterWorkLogStatistic.ts
export class FilterWorkLogStatistic {

  avgTemperature: number;
  sumFilterContamination: number;
  scanDate: string;

  constructor(avgTemperature: number, sumFilterContamination: number, scanDate: string) {
    this.avgTemperature = avgTemperature;
    this.sumFilterContamination = sumFilterContamination;
    this.scanDate = scanDate;
  }

}
